### PR TITLE
Expire post

### PR DESCRIPTION
**Pull request by HardysJin**
### Check list

 - [x] The branch name follows the naming convention.
 - [x] Tests have been written.
 - [x] The code has a standard coding style or fixed using `yarn lint`.
 - [x] The pull request is attached in [Trello board](https://trello.com/b/pPtDOXkb).

### CODE REVIEW

**Comment by HardysJin** [comment]
[Expire post](https://trello.com/c/r7zdRp4I/16-expire-post)

File: src/inputs/Post.input.ts [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 @InputType({ description: "Create post data" })
 export class PostInput {
     @Field()
-    @IsNotEmpty()
+    @Length(1, 255)
     content: string;
+
+    @Field(returns => Int, {
+        nullable: true,
+        description: "nullable, integers that stores unix timestamp",
```
Comment (senhungwong) [inline src/inputs/Post.input.ts]: Description is not necessary

File: src/resolvers/PostResolver.ts [hunk 2]
```diff
@@ -11,6 +12,10 @@ export class PostResolver {
         @Arg("pagination", { defaultValue: new PaginationInput() }) pagination: PaginationInput,
     ): Promise<Post[]> {
         const posts = await Post.find({
+            where: [
+                { expiredAt: MoreThan(Date.now() / 1000 | 0) },
```
Comment (senhungwong) [inline src/resolvers/PostResolver.ts]: `Date.now()` will be computed multiple times here.

```
const currentTimestamp = Date.now() / 1000 | 0;

const posts = await Post.find({ ... })
```

File: src/inputs/Post.input.ts (hunk 1 above)
Comment (HardysJin) [reply src/inputs/Post.input.ts]: Removed

File: src/resolvers/PostResolver.ts (hunk 2 above)
Comment (HardysJin) [reply src/resolvers/PostResolver.ts]: Fixed.

**Pull request merged**